client/src/App.js

```diff
@@ -88,6 +90,24 @@ export default function App() {
       ...series.slice(index + 1),
     ])
   }
+
+  function handleCheckEpisode(id) {
+    const checkIfOnList = watchedEpisodes.some(el => el === id)
+
+    if (checkIfOnList) {
+      const index = watchedEpisodes.findIndex(el => el === id)
+      setWatchedEpisodes([
+        ...watchedEpisodes.slice(0, index),
+        ...watchedEpisodes.slice(index + 1),
+      ])
+    } else {
+      setWatchedEpisodes([...watchedEpisodes, id])
+    }
+  }
```

Comment:
It took me a bit to figure out what you are doing here, probably because it's late 😅.
I did get confused with the wording though. `id` is one episode, right? Maybe call it `episode` then as I though: "Why is he adding the `id` to the watched episodes?"
And `checkIfOnList` sounds like a function but is a bool, maybe consider calling it `isOnList`? So that it is yet a bit clearer. 
```suggestion
  function handleCheckEpisode(episode) {
    const isOnList = watchedEpisodes.some(el => el === episode)

    if (isOnList) {
      const index = watchedEpisodes.findIndex(el => el === episode)
      setWatchedEpisodes([
        ...watchedEpisodes.slice(0, index),
        ...watchedEpisodes.slice(index + 1),
      ])
    } else {
      setWatchedEpisodes([...watchedEpisodes, episode])
    }
  }
```

Reply:
Or is `watchedEpisodes` an array of `id`s? I am a bit confused. 😄

Reply:
Yes, you are absolutely right. It's confusing. I changed the wording to make it more clear and yes the `watchedEpisodes` is an array of `id`s.